Bottom: 69d15294161e589ef106168550b5028b765dda01
Top:    d3977303ea4703c5b10ca9e45419e87117e07680
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-08 17:33:52 +0100

Refresh of attempt-to-refactor-the-step

---

diff --git a/src/hid/gtk/step.c b/src/hid/gtk/step.c
index dfb1deb..090a338 100644
--- a/src/hid/gtk/step.c
+++ b/src/hid/gtk/step.c
@@ -180,7 +180,7 @@ step_face_bound (step_file *file, char *name, step_id bound, bool orientation)
 step_id
 step_face_outer_bound (step_file *file, char *name, step_id bound, bool orientation)
 {
-  fprintf (file->f, "#%i = FACE_BOUND ( '%s', #%i, %s ) ;\n",
+  fprintf (file->f, "#%i = FACE_OUTER_BOUND ( '%s', #%i, %s ) ;\n",
                     file->next_id, name, bound, step_bool (orientation));
 
   return file->next_id++;
